mail/cclient/Portfile

```diff
@@ -65,6 +65,11 @@ if {[vercmp ${os.version} 10] >= 0} {
     patchfiles-append patch-Makefile-fixpam
 }
 
+if { ![catch {set vers [lindex [registry_active openssl] 0]}]
+     && ([vercmp [lindex $vers 1] 1.1] >= 0) } {
+    patchfiles-append patch-03-openssl_1_1
+}
+
```

Comment:
I'm not understanding why you are using a conditional to apply the patch? This patch should always be applied given that the openssl port is at version 1.1.

Reply:
I wasn't sure whether this was needed; thanks for clarifying!